To: vim-dev@vim.org
Subject: Patch 7.3.022
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.022
Problem:    When opening a new window the 'spellcapcheck' option is cleared.
Solution:   Copy the correct option value. (Christian Brabandt)
Files:	    src/option.c


*** ../vim-7.3.021/src/option.c	2010-08-15 21:57:28.000000000 +0200
--- src/option.c	2010-10-13 13:48:46.000000000 +0200
***************
*** 10011,10017 ****
  	    buf->b_p_smc = p_smc;
  #endif
  #ifdef FEAT_SPELL
! 	    buf->b_s.b_p_spc = vim_strsave(p_spf);
  	    (void)compile_cap_prog(&buf->b_s);
  	    buf->b_s.b_p_spf = vim_strsave(p_spf);
  	    buf->b_s.b_p_spl = vim_strsave(p_spl);
--- 10011,10017 ----
  	    buf->b_p_smc = p_smc;
  #endif
  #ifdef FEAT_SPELL
! 	    buf->b_s.b_p_spc = vim_strsave(p_spc);
  	    (void)compile_cap_prog(&buf->b_s);
  	    buf->b_s.b_p_spf = vim_strsave(p_spf);
  	    buf->b_s.b_p_spl = vim_strsave(p_spl);
*** ../vim-7.3.021/src/version.c	2010-10-10 17:08:28.000000000 +0200
--- src/version.c	2010-10-13 14:02:08.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     22,
  /**/

-- 
GALAHAD:   Camelot ...
LAUNCELOT: Camelot ...
GAWAIN:    It's only a model.
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\        download, build and distribute -- http://www.A-A-P.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
